### PR TITLE
Fix debug tools showing 'wrappedComponent' for devTools, fixes #73

### LINKED ISSUE #73: Using const MyComponent = observer((props) => to create observer component makes unnamed component <Memo(wrappedComponent)>  in React DevTools
I know useObserver is the preferred approach but still, sometimes observer is needed, is this a known issue are there plans to get the proper name instead of this?
image
image

### PR DESCRIPTION
**Pull request by mweststrate**
React.memo's own `displayName` is not used by the React devtools, rather, the name of the component being wrapped is used. This PR gives the generated `wrappedComponent` the right name, which fixes #73

### CODE REVIEW

**Comment by mweststrate** [comment]
@FredyC no idea why the build fails, seems unrelated to the change

**Comment by xaviergonz** [comment]
node 11.11 made some changes to how process is mangled, the newest version of jest fixes it

**Comment by xaviergonz** [comment]
TLDR: upgrade jest :)
(and check MST pull requests whenever you have the time if possible ;) )

**Comment by xaviergonz** [comment]
@mweststrate fixed it for you

**Comment by mweststrate** [comment]
@xaviergonz thanks! Sorry, I'm really behind with PR reviews, trying to catch up, but I only have very little time till april :(

**Comment by xaviergonz** [comment]
@mweststrate no problem! but if you could answer the question posed on https://github.com/mobxjs/mobx-state-tree/pull/1205 (no need to read the actual code) that'd be great :)

**Comment by FredyC** [comment]
Thanks @mweststrate, haven't thought there is an actual bug in here :)
